features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -205,7 +205,9 @@ internal class SessionReplayFeature(
             return
         }
 
-        if (keepSession && rateBasedSampler.sample()) {
+        val shouldRecord = startRecordingImmediately || isRecording.get()
```

Comment:
where is `isRecording.get` coming from and why it is used here ?

Reply:
It's a flag in this class that we set to true when recordings are manually started (and set to false when recordings are manually stopped)